interface/super/edit_list.php

```diff
@@ -1058,6 +1088,49 @@ function mysubmit() {
                     </select>
                 </div>
 
+                <!--Added filter-->
+                <script type="text/javascript">
+                    function lister() {
+                        var urlFull   = new URL($(location).attr('href'));
+                        var list_from = parseInt($("#list-from").val());
+                        var list_to   = parseInt($("#list-to").val());
+                        var list_id_container = $("#list_id").val();
+
+                        if( list_from > list_to ){
+                            alert(<?php echo xlj("Please enter a enter valid range"); ?>);
+                            return false;
+                        }
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_from");
+                            urlFull.searchParams.set("list_from",list_from);
+                        }
+
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_to");
+                            urlFull.searchParams.set("list_to",list_to);
+                        }
+                        if( list_id_container.length > 0 ){
+                            urlFull.searchParams.delete("list_id_container");
+                            urlFull.searchParams.set("list_id_container", list_id_container );
+                        }
+                        window.location.replace(urlFull);
+                    }
+                </script>
+                <?php
+                    $urlFrom = ( isset($_REQUEST["list_from"]) && intval($_REQUEST["list_from"]) >= 0 ? intval($_REQUEST["list_from"]) :  1 );
+                    $urlTo   = ( isset($_REQUEST["list_to"]) && intval($_REQUEST["list_to"]) >= 0 ? intval($_REQUEST["list_to"]) : '' );
+                    if( $urlTo == 0){
+                        $urlTo = '';
+                    }
+                ?>
+                <div class="blck-filter" style="float: left; margin-top: 5px; margin-left: 10px; border:0px solid red; width: auto; ">
```

Comment:
Why is there inline CSS here? Our themes CSS should be able to handle this

Reply:
Hi @robertdown . I use inline CSS, because I need to know where I can add CSS style for this div.

Reply:
I’ll put down this code to get a look at the UI and how to best get it incorporated into our core CSS. Inline CSS is something we are trying to get out of the codebase (it creates too many non-consistent UI experiences)